### PR TITLE
en,zh: add v1.5.4 release notes

### PR DESCRIPTION
**Pull request by csuzhangxc**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.6 (TiDB Operator 1.6 versions)
- [x] v1.5 (TiDB Operator 1.5 versions)
- [ ] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: en/releases/release-1.5.4.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Release date: September 13, 2024
+
+TiDB Operator version: 1.5.4
+
+## Improvements
+
+- VolumeReplace feature supports customizing the number of spare replicas used during TiKV or PD disk replacements ([#5666](https://github.com/pingcap/tidb-operator/pull/5666), [@anish-db](https://github.com/anish-db))
+- VolumeReplace feature supports only enabling for specified TiDB clusters ([#5670](https://github.com/pingcap/tidb-operator/pull/5670), [@rajsuvariya](https://github.com/rajsuvariya))
+
```
Comment (BornChanger) [inline en/releases/release-1.5.4.md:16]: - EBS Snapshot restore supports to configure if warmup failure on individual volumes fails the restore instantly ([#5635](https://github.com/pingcap/tidb-operator/pull/5635), [@michealdeng](https://github.com/[michaelmdeng](https://github.com/michaelmdeng)))
- Gate restore failure on warmup failure only for `check-wal-only` warmup strategy. ([#5636](https://github.com/pingcap/tidb-operator/pull/5636), [@michealdeng](https://github.com/[michaelmdeng](https://github.com/michaelmdeng)))
- support to fetch backup size from backupmeta ([#5430](https://github.com/pingcap/tidb-operator/pull/5430), [@Leavrth](https://github.com/Leavrth))

File: zh/releases/release-1.5.4.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+TiDB Operator 版本：1.5.4
+
+## 优化提升
+
+- VolumeReplace 功能支持为 PD 和 TiKV 自定义备用副本数量 ([#5666](https://github.com/pingcap/tidb-operator/pull/5666), [@anish-db](https://github.com/anish-db))
+- VolumeReplace 功能支持仅为特定 TiDB 集群开启 ([#5670](https://github.com/pingcap/tidb-operator/pull/5670), [@rajsuvariya](https://github.com/rajsuvariya))
+
+## Bug 修复
+
+- 修复 tidb-backup-manager 无法解析 BR backupmeta v2 中备份文件存储大小的问题 ([#5411](https://github.com/pingcap/tidb-operator/pull/5411), [@Leavrth](https://github.com/Leavrth))
```
Comment (BornChanger) [inline zh/releases/release-1.5.4.md:19]:
```suggestion
- 修复 tidb-backup-manager 无法解析 BR backupmeta v2 中备份文件存储大小的问题 ([#5411](https://github.com/pingcap/tidb-operator/pull/5411), [@Leavrth](https://github.com/Leavrth))
- 支持 EBS snapshot volume 恢复失败时，清理 EBS 卷的能力 ([#5639](https://github.com/pingcap/tidb-operator/pull/5639), [@WangLe1321](https://github.com/WangLe1321))
- federated manager 重启后，重置相关的 metrics 指标 ([#5648](https://github.com/pingcap/tidb-operator/pull/5648), [@wxiaomou](https://github.com/wxiaomou))
- EBS Snapshot 备份增强对 TC 检查，如果集群没有配置 TiKV 节点或者 TiKV replica 为0的时候，让备份失败 ([#5662](https://github.com/pingcap/tidb-operator/pull/5662), [@BornChanger ](https://github.com/BornChanger))
```

File: zh/releases/release-1.5.4.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+发布日期：2024 年 9 月 13 日
+
+TiDB Operator 版本：1.5.4
+
+## 优化提升
+
+- VolumeReplace 功能支持为 PD 和 TiKV 自定义备用副本数量 ([#5666](https://github.com/pingcap/tidb-operator/pull/5666), [@anish-db](https://github.com/anish-db))
+- VolumeReplace 功能支持仅为特定 TiDB 集群开启 ([#5670](https://github.com/pingcap/tidb-operator/pull/5670), [@rajsuvariya](https://github.com/rajsuvariya))
+
```
Comment (BornChanger) [inline zh/releases/release-1.5.4.md:16]:
```suggestion
- EBS 快照恢复支持设定某个卷 warmup 失败是否立刻整个恢复作业 ([#5635](https://github.com/pingcap/tidb-operator/pull/5635), [@michealdeng](https://github.com/[michaelmdeng](https://github.com/michaelmdeng)))
- EBS 快照恢复在使用 `check-wal-only` 策略时，warmup 失败会将整个 restore 失败 ([#5636](https://github.com/pingcap/tidb-operator/pull/5636), [@michealdeng](https://github.com/[michaelmdeng](https://github.com/michaelmdeng)))
```

File: en/releases/release-1.5.4.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+## Bug fixes
+
+- Fix the issue that tidb-backup-manager can't parse files storage size from BR backupmeta v2 ([#5411](https://github.com/pingcap/tidb-operator/pull/5411), [@Leavrth](https://github.com/Leavrth))
```
Comment (BornChanger) [inline en/releases/release-1.5.4.md:19]:
```suggestion
- Fix the issue that tidb-backup-manager can't parse files storage size from BR backupmeta v2 ([#5411](https://github.com/pingcap/tidb-operator/pull/5411), [@Leavrth](https://github.com/Leavrth))
- cleanup EBS volumes if EBS snapshot restore fails ([#5639](https://github.com/pingcap/tidb-operator/pull/5639), [@WangLe1321](https://github.com/WangLe1321))
- Reset metrics when federated manager restarts ([#5648](https://github.com/pingcap/tidb-operator/pull/5648), [@wxiaomou](https://github.com/wxiaomou))
- Enhance TC check for EBS Snapshot and fail EBS snapshot backup if no TiKV nodes or replica is 0 ([#5662](https://github.com/pingcap/tidb-operator/pull/5662), [@BornChanger ](https://github.com/BornChanger))
```

File: zh/releases/release-1.5.4.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+发布日期：2024 年 9 月 13 日
+
+TiDB Operator 版本：1.5.4
+
+## 优化提升
+
+- VolumeReplace 功能支持为 PD 和 TiKV 自定义备用副本数量 ([#5666](https://github.com/pingcap/tidb-operator/pull/5666), [@anish-db](https://github.com/anish-db))
+- VolumeReplace 功能支持仅为特定 TiDB 集群开启 ([#5670](https://github.com/pingcap/tidb-operator/pull/5670), [@rajsuvariya](https://github.com/rajsuvariya))
+- EBS 快照恢复支持设定某个卷 warmup 失败是否立刻让整个恢复作业失败 ([#5622](https://github.com/pingcap/tidb-operator/pull/5622), [@michaelmdeng](https://github.com/michaelmdeng))
+- EBS 快照恢复在使用 `check-wal-only` 策略时，warmup 失败会将整个 restore 设置为失败 ([#5621](https://github.com/pingcap/tidb-operator/pull/5621), [@michaelmdeng](https://github.com/michaelmdeng))
```
Comment (Oreoxmt) [inline zh/releases/release-1.5.4.md:17]:
```suggestion
- EBS 快照恢复支持设定某个卷 warmup 失败时是否立即终止整个恢复任务 ([#5622](https://github.com/pingcap/tidb-operator/pull/5622), [@michaelmdeng](https://github.com/michaelmdeng))
- EBS 快照恢复在使用 `check-wal-only` 策略时，如果 warmup 失败，整个恢复任务将被设置为失败 ([#5621](https://github.com/pingcap/tidb-operator/pull/5621), [@michaelmdeng](https://github.com/michaelmdeng))
```

File: zh/releases/release-1.5.4.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+## Bug 修复
+
+- 修复 tidb-backup-manager 无法解析 BR backupmeta v2 中备份文件存储大小的问题 ([#5411](https://github.com/pingcap/tidb-operator/pull/5411), [@Leavrth](https://github.com/Leavrth))
+- 修复 EBS snapshot restore 失败时，EBS 卷可能泄露的问题 ([#5634](https://github.com/pingcap/tidb-operator/pull/5634), [@WangLe1321](https://github.com/WangLe1321))
+- 修复 Federated manager 重启后，未正确初始化相关的 metrics 指标的问题 ([#5637](https://github.com/pingcap/tidb-operator/pull/5637), [@wxiaomou](https://github.com/wxiaomou))
+- 修复没有配置 TiKV 节点或者 TiKV replica 为 0 的时，EBS Snapshot 未能失败终止的问题 ([#5659](https://github.com/pingcap/tidb-operator/pull/5659), [@BornChanger](https://github.com/BornChanger))
```
Comment (Oreoxmt) [inline zh/releases/release-1.5.4.md:24]:
```suggestion
- 修复 EBS 快照恢复失败时，EBS 卷可能泄露的问题 ([#5634](https://github.com/pingcap/tidb-operator/pull/5634), [@WangLe1321](https://github.com/WangLe1321))
- 修复 Federated manager 重启后未正确初始化相关指标的问题 ([#5637](https://github.com/pingcap/tidb-operator/pull/5637), [@wxiaomou](https://github.com/wxiaomou))
- 修复在没有配置 TiKV 节点或者 TiKV 副本数为 0 时，EBS 快照备份失败的问题 ([#5659](https://github.com/pingcap/tidb-operator/pull/5659), [@BornChanger](https://github.com/BornChanger))
```

Comment (csuzhangxc) [reply zh/releases/release-1.5.4.md:24]: 最后那条，其实是本来应该没有 TiKV 就是要失败的，但在这个修复之前，也错误地成功了

Comment (Oreoxmt) [reply zh/releases/release-1.5.4.md:24]: @csuzhangxc 已更新为“EBS 快照备份无法终止的问题”

File: en/releases/release-1.5.4.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+Release date: September 13, 2024
+
+TiDB Operator version: 1.5.4
+
+## Improvements
+
+- VolumeReplace feature supports customizing the number of spare replicas used during TiKV or PD disk replacements ([#5666](https://github.com/pingcap/tidb-operator/pull/5666), [@anish-db](https://github.com/anish-db))
+- VolumeReplace feature supports only enabling for specified TiDB clusters ([#5670](https://github.com/pingcap/tidb-operator/pull/5670), [@rajsuvariya](https://github.com/rajsuvariya))
+- EBS Snapshot restore supports to configure if warmup failure on individual volume fails the restore instantly ([#5622](https://github.com/pingcap/tidb-operator/pull/5622), [@michaelmdeng](https://github.com/michaelmdeng))
+- Gate EBS Snapshot restore failure on warmup failure only for `check-wal-only` warmup strategy ([#5621](https://github.com/pingcap/tidb-operator/pull/5621), [@michaelmdeng](https://github.com/michaelmdeng))
```
Comment (Oreoxmt) [inline en/releases/release-1.5.4.md:17]:
```suggestion
- The VolumeReplace feature supports customizing the number of spare replicas for PD and TiKV ([#5666](https://github.com/pingcap/tidb-operator/pull/5666), [@anish-db](https://github.com/anish-db))
- The VolumeReplace feature can be enabled for specific TiDB clusters ([#5670](https://github.com/pingcap/tidb-operator/pull/5670), [@rajsuvariya](https://github.com/rajsuvariya))
- EBS snapshot restore supports configuring whether to terminate the entire restore task immediately if volume warmup fails ([#5622](https://github.com/pingcap/tidb-operator/pull/5622), [@michaelmdeng](https://github.com/michaelmdeng))
- When using the `check-wal-only` warmup strategy, EBS snapshot restore marks the entire restore task as failed if warmup fails ([#5621](https://github.com/pingcap/tidb-operator/pull/5621), [@michaelmdeng](https://github.com/michaelmdeng))
```

File: en/releases/release-1.5.4.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+## Bug fixes
+
+- Fix the issue that tidb-backup-manager can't parse files storage size from BR backupmeta v2 ([#5411](https://github.com/pingcap/tidb-operator/pull/5411), [@Leavrth](https://github.com/Leavrth))
+- Fix the issue that EBS may leak after EBS snapshot restore fails ([#5634](https://github.com/pingcap/tidb-operator/pull/5634), [@WangLe1321](https://github.com/WangLe1321))
+- Fix the issue that the metrics are not initialized after the federated manager restarts ([#5637](https://github.com/pingcap/tidb-operator/pull/5637), [@wxiaomou](https://github.com/wxiaomou))
+- Fix the issue that EBS snapshot restore doesn't fail even if no TiKV instance configured or replica is set to 0 ([#5659](https://github.com/pingcap/tidb-operator/pull/5659), [@BornChanger](https://github.com/BornChanger))
```
Comment (Oreoxmt) [inline en/releases/release-1.5.4.md:24]:
```suggestion
- Fix the issue that tidb-backup-manager cannot parse backup file storage size in BR backupmeta v2 ([#5411](https://github.com/pingcap/tidb-operator/pull/5411), [@Leavrth](https://github.com/Leavrth))
- Fix a potential EBS volume leak issue when EBS snapshot restore fails ([#5634](https://github.com/pingcap/tidb-operator/pull/5634), [@WangLe1321](https://github.com/WangLe1321))
- Fix the issue that metrics are not properly initialized after federated manager restarts ([#5637](https://github.com/pingcap/tidb-operator/pull/5637), [@wxiaomou](https://github.com/wxiaomou))
- Fix the issue that EBS snapshot restore does not fail when no TiKV instances are configured or TiKV replica is set to 0 ([#5659](https://github.com/pingcap/tidb-operator/pull/5659), [@BornChanger](https://github.com/BornChanger))
```

File: zh/releases/release-1.5.4.md (hunk 6 above)
Comment (csuzhangxc) [reply zh/releases/release-1.5.4.md:24]: 好像 无法终止 和 错误地成功 还是有点细微的差别。修复之前的问题是，本来应该是 失败，但任务最后的结果是说 成功了

File: en/releases/release-1.5.4.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+- When using the `check-wal-only` warmup strategy, EBS snapshot restore marks the entire restore task as failed if warmup fails ([#5621](https://github.com/pingcap/tidb-operator/pull/5621), [@michaelmdeng](https://github.com/michaelmdeng))
+
+## Bug fixes
+
+- Fix the issue that tidb-backup-manager cannot parse backup file storage size in BR backupmeta v2 ([#5411](https://github.com/pingcap/tidb-operator/pull/5411), [@Leavrth](https://github.com/Leavrth))
+- Fix a potential EBS volume leak issue when EBS snapshot restore fails ([#5634](https://github.com/pingcap/tidb-operator/pull/5634), [@WangLe1321](https://github.com/WangLe1321))
+- Fix the issue that metrics are not properly initialized after federated manager restarts ([#5637](https://github.com/pingcap/tidb-operator/pull/5637), [@wxiaomou](https://github.com/wxiaomou))
+- Fix the issue that EBS snapshot restore does not fail when no TiKV instances are configured or TiKV replica is set to 0 ([#5659](https://github.com/pingcap/tidb-operator/pull/5659), [@BornChanger](https://github.com/BornChanger))
```
Comment (Oreoxmt) [inline en/releases/release-1.5.4.md:24]:
```suggestion
- Fix the issue that EBS snapshot restore incorrectly succeeds when no TiKV instances are configured or TiKV replica is set to 0 ([#5659](https://github.com/pingcap/tidb-operator/pull/5659), [@BornChanger](https://github.com/BornChanger))
```

Comment (Oreoxmt) [reply en/releases/release-1.5.4.md:24]: @csuzhangxc PTAL agin

File: zh/releases/release-1.5.4.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
+---
+title: TiDB Operator 1.5.4 Release Notes
+summary: 了解 TiDB Operator 1.5.4 版本的新功能和 Bug 修复。
```
Comment (qiancai) [inline zh/releases/release-1.5.4.md:3]:
```suggestion
summary: 了解 TiDB Operator 1.5.4 版本的优化提升和 Bug 修复。
```

File: en/releases/release-1.5.4.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
+---
+title: TiDB Operator 1.5.4 Release Notes
+summary: Learn about new features and bug fixes in TiDB Operator 1.5.4.
```
Comment (qiancai) [inline en/releases/release-1.5.4.md:3]:
```suggestion
summary: Learn about improvements and bug fixes in TiDB Operator 1.5.4.
```

File: zh/releases/release-1.5.4.md [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+## 优化提升
+
+- VolumeReplace 功能支持为 PD 和 TiKV 自定义备用副本数量 ([#5666](https://github.com/pingcap/tidb-operator/pull/5666), [@anish-db](https://github.com/anish-db))
+- VolumeReplace 功能支持仅为特定 TiDB 集群开启 ([#5670](https://github.com/pingcap/tidb-operator/pull/5670), [@rajsuvariya](https://github.com/rajsuvariya))
+- EBS 快照恢复支持设定某个卷 warmup 失败时是否立即终止整个恢复任务 ([#5622](https://github.com/pingcap/tidb-operator/pull/5622), [@michaelmdeng](https://github.com/michaelmdeng))
+- EBS 快照恢复在使用 `check-wal-only` 策略时，如果 warmup 失败，整个恢复任务将被设置为失败 ([#5621](https://github.com/pingcap/tidb-operator/pull/5621), [@michaelmdeng](https://github.com/michaelmdeng))
+
+## Bug 修复
+
+- 修复 tidb-backup-manager 无法解析 BR backupmeta v2 中备份文件存储大小的问题 ([#5411](https://github.com/pingcap/tidb-operator/pull/5411), [@Leavrth](https://github.com/Leavrth))
```
Comment (qiancai) [inline zh/releases/release-1.5.4.md:21]:
```suggestion
- 修复 `tidb-backup-manager` 无法解析 BR backupmeta v2 中备份文件大小的问题 ([#5411](https://github.com/pingcap/tidb-operator/pull/5411), [@Leavrth](https://github.com/Leavrth))
```

File: en/releases/release-1.5.4.md [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+## Improvements
+
+- The VolumeReplace feature supports customizing the number of spare replicas for PD and TiKV ([#5666](https://github.com/pingcap/tidb-operator/pull/5666), [@anish-db](https://github.com/anish-db))
+- The VolumeReplace feature can be enabled for specific TiDB clusters ([#5670](https://github.com/pingcap/tidb-operator/pull/5670), [@rajsuvariya](https://github.com/rajsuvariya))
+- EBS snapshot restore supports configuring whether to terminate the entire restore task immediately if volume warmup fails ([#5622](https://github.com/pingcap/tidb-operator/pull/5622), [@michaelmdeng](https://github.com/michaelmdeng))
+- When using the `check-wal-only` warmup strategy, EBS snapshot restore marks the entire restore task as failed if warmup fails ([#5621](https://github.com/pingcap/tidb-operator/pull/5621), [@michaelmdeng](https://github.com/michaelmdeng))
+
+## Bug fixes
+
+- Fix the issue that tidb-backup-manager cannot parse backup file storage size in BR backupmeta v2 ([#5411](https://github.com/pingcap/tidb-operator/pull/5411), [@Leavrth](https://github.com/Leavrth))
```
Comment (qiancai) [inline en/releases/release-1.5.4.md:21]:
```suggestion
- Fix the issue that `tidb-backup-manager` cannot parse backup file size in BR backupmeta v2 ([#5411](https://github.com/pingcap/tidb-operator/pull/5411), [@Leavrth](https://github.com/Leavrth))
```

File: en/releases/release-1.5.4.md [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+- Fix a potential EBS volume leak issue when EBS snapshot restore fails ([#5634](https://github.com/pingcap/tidb-operator/pull/5634), [@WangLe1321](https://github.com/WangLe1321))
```
Comment (qiancai) [inline en/releases/release-1.5.4.md:22]:
```suggestion
- Fix a potential EBS volume leak issue that occurs when EBS snapshot restore fails ([#5634](https://github.com/pingcap/tidb-operator/pull/5634), [@WangLe1321](https://github.com/WangLe1321))
```

File: zh/releases/release-1.5.4.md [hunk 15]
```diff
@@ -0,0 +1,24 @@
...
+- 修复 EBS 快照恢复失败时，EBS 卷可能泄露的问题 ([#5634](https://github.com/pingcap/tidb-operator/pull/5634), [@WangLe1321](https://github.com/WangLe1321))
+- 修复 Federated manager 重启后未正确初始化相关指标的问题 ([#5637](https://github.com/pingcap/tidb-operator/pull/5637), [@wxiaomou](https://github.com/wxiaomou))
+- 修复在没有配置 TiKV 节点或者 TiKV 副本数为 0 时，EBS 快照恢复错误地成功的问题 ([#5659](https://github.com/pingcap/tidb-operator/pull/5659), [@BornChanger](https://github.com/BornChanger))
```
Comment (qiancai) [inline zh/releases/release-1.5.4.md:24]:
```suggestion
- 修复在没有配置 TiKV 节点或者 TiKV 副本数为 0 时，EBS 快照恢复错误地显示为成功的问题 ([#5659](https://github.com/pingcap/tidb-operator/pull/5659), [@BornChanger](https://github.com/BornChanger))
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**